[win32] add prepare-addons-dev.bat for easier binary addon development

Montellese opened the pull request:
`prepare-addons-dev.bat` is a win32 helper script I wrote for the PVR devs. It is the development equivalent to the `make-addons.bat` which we use for releases.

The key differences are
- `CMAKE_BUILD_TYPE` is set to `Debug` and not `Release`
- it uses the `Visual Studio 12` generator instead of `NMake Makefiles`
- it does not actually build the binary addons but only prepares the Visual Studio based build environment

Montellese committed:
[win32] add prepare-binary-addons-dev.bat for easier binary addon development

Jalle19 commented:
Thanks! Any chance you could add some switch to override the addon source directory, that way you wouldn't have to add/modify the addon .txt files. It's a big deal so don't bother if it makes the script too complicated.

Memphiz commented:
mhhh the tools/buildsteps dir was ment for CI scripts at one point - but i figured that its a bit merged for windows anyways. No hard objections here - its just how it was thought out by cory iirc.

Montellese commented:
I can put it somewhere else. I'd just have to adjust the paths in the script.

Memphiz commented:
Well if it is not used via jenkins i tend to like that idea ;)

Montellese commented:
Where should I put it? Can't think of any place where it would really belong :-/

Jalle19 commented:
project/cmake/addons? That's were I put it before I realized it was the wrong place.

Montellese commented:
I updated it with the newest features from `make-addons.bat` and I moved it to `tools/windows/prepare-binary-addons-dev.bat` to keep the `tools/buildsteps/win32` and `project/cmake/addons` directories clean.
